crates/ruff_linter/src/rules/flake8_pie/snapshots/ruff_linter__rules__flake8_pie__tests__preview__PIE790_PIE790.py.snap

```diff
@@ -0,0 +1,653 @@
+141     |-    pass
+142 141 | 
+143 142 | for i in range(10):
+144 143 |     pass
+
+PIE790.py:141:5: PIE790 [*] Unnecessary `pass` statement
+    |
+139 | for i in range(10):
+140 |     pass
+141 |     pass
+    |     ^^^^ PIE790
+142 | 
+143 | for i in range(10):
+    |
+    = help: Remove unnecessary `pass`
+
+ℹ Safe fix
+138 138 | 
+139 139 | for i in range(10):
+140 140 |     pass
+141     |-    pass
+142 141 | 
+143 142 | for i in range(10):
+144 143 |     pass
+
+PIE790.py:144:5: PIE790 [*] Unnecessary `pass` statement
+    |
+143 | for i in range(10):
+144 |     pass
+    |     ^^^^ PIE790
+145 | 
+146 |     pass
+    |
+    = help: Remove unnecessary `pass`
+
+ℹ Safe fix
+141 141 |     pass
+142 142 | 
+143 143 | for i in range(10):
+144     |-    pass
+145 144 | 
+146 145 |     pass
+147 146 | 
+
+PIE790.py:146:5: PIE790 [*] Unnecessary `pass` statement
+    |
+144 |     pass
+145 | 
+146 |     pass
+    |     ^^^^ PIE790
+147 | 
+148 | for i in range(10):
+    |
+    = help: Remove unnecessary `pass`
+
+ℹ Safe fix
+143 143 | for i in range(10):
+144 144 |     pass
+145 145 | 
+146     |-    pass
+147 146 | 
+148 147 | for i in range(10):
+149 148 |     pass  # comment
+
+PIE790.py:149:5: PIE790 [*] Unnecessary `pass` statement
+    |
+148 | for i in range(10):
+149 |     pass  # comment
+    |     ^^^^ PIE790
+150 |     pass
+    |
+    = help: Remove unnecessary `pass`
+
+ℹ Safe fix
+146 146 |     pass
+147 147 | 
+148 148 | for i in range(10):
+149     |-    pass  # comment
+    149 |+    # comment
+150 150 |     pass
+151 151 | 
+152 152 | 
+
+PIE790.py:150:5: PIE790 [*] Unnecessary `pass` statement
+    |
+148 | for i in range(10):
+149 |     pass  # comment
+150 |     pass
+    |     ^^^^ PIE790
+    |
+    = help: Remove unnecessary `pass`
+
+ℹ Safe fix
+147 147 | 
+148 148 | for i in range(10):
+149 149 |     pass  # comment
+150     |-    pass
+151 150 | 
+152 151 | 
+153 152 | def foo():
+
+PIE790.py:155:5: PIE790 [*] Unnecessary `...` expression
+    |
+153 | def foo():
+154 |     print("foo")
+155 |     ...
+    |     ^^^ PIE790
+    |
+    = help: Remove unnecessary `...`
+
+ℹ Safe fix
+152 152 | 
+153 153 | def foo():
+154 154 |     print("foo")
+155     |-    ...
+156 155 | 
+157 156 | 
+158 157 | def foo():
+
+PIE790.py:161:5: PIE790 [*] Unnecessary `...` expression
+    |
+159 |     """A docstring."""
+160 |     print("foo")
+161 |     ...
+    |     ^^^ PIE790
+    |
+    = help: Remove unnecessary `...`
+
+ℹ Safe fix
+158 158 | def foo():
+159 159 |     """A docstring."""
+160 160 |     print("foo")
+161     |-    ...
+162 161 | 
+163 162 | 
+164 163 | for i in range(10):
+
+PIE790.py:165:5: PIE790 [*] Unnecessary `...` expression
+    |
+164 | for i in range(10):
+165 |     ...
+    |     ^^^ PIE790
+166 |     ...
+    |
+    = help: Remove unnecessary `...`
+
+ℹ Safe fix
+163 163 | 
+164 164 | for i in range(10):
+165 165 |     ...
+166     |-    ...
+167 166 | 
+168 167 | for i in range(10):
+169 168 |     ...
```

Comment:
I'm not sure if this is a bug in the diff printer or the range is incorrect but the violation is on line 165 while the fix is removing the `...` literal from the _next_ line (166).